src/auth/strategies/strategies.conts.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Different auth strategies available
+ */
+export enum AuthStrategy {
+  // note - order here is the evaluation order, its not arbitrary
+  apiKey = 'apiKey',
+  open = 'open',
+}
+
+export const authStrategyValues = Object.values(AuthStrategy);
+
+/**
+ * A helper to be passed to `.sort`. The order is in which they will be evaluated.
+ *
+ * For example the the "open" strategy should always be last to allow for testing of other strategies
+ */
+export const cmpAuthStrategyOrder = (a: AuthStrategy, b: AuthStrategy): number =>
```

Comment:
I don't want someone to write `open,apiKey` config and wonder why apiKey never gets executed.